### DISCUSSION TITLE
How to set the last MontlyPartition to the current month

### DISCUSSION THREAD

**Discussion by dridk** [dagster-io/dagster#24094] [Q&A] [answered]
I have an asset partitioned by month  where I only define the start_date.  We are in August and the last partition proposed in the Dagit interface is July. Is it possible to configure the asset so that dagit proposes the current month as the last partition?

**Comment by garethbrickman** [comment, marked as answer]
I think this is maybe something like what you are looking for?

> end_offset (int) – Extends the partition set by a number of partitions equal to the value passed. If end_offset is 0 (the default), the last partition ends before the current time. If end_offset is 1, the second-to-last partition ends before the current time, and so on.

https://docs.dagster.io/_apidocs/partitions#dagster.MonthlyPartitionsDefinition

**Reply by dridk** [reply]
Awesome!! Tks!!!!
